lib/util/init.git.js

```diff
@@ -252,30 +252,10 @@ const Init = {
      * @returns {Promise.<{'user.name': string, 'user.email': string}>} user.name and user.email
      */
     async _getGitConfigUser() {
-        const gitConfigs = await git.listConfig();
-        // remove local config
-        delete gitConfigs.values['.git/config'];
-        const result = {};
+        const name = Util.execSync('git', ['config', 'user.name'], true);
```

Comment:
kinda confused what the objective is here? you are moving away from the helper lib to using native commands, but not sure why?

Reply:
if we run the direct commands it automatically takes the local config first, then reads the local one. and it returns whatever value is in that config last. 
simple-git tried to be better but did it worse as a result. if there were multiple entries, it would return an array, even though that didn't make sense and git wouldn't care about that list but only use the last entry

Reply:
alright. there was actually another command that resolved it like calling git via CLI does. i pushed the change and now we are back to using simple-git - but in a smarter way than before at least.